### PR TITLE
Configure log format

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -32,7 +36,13 @@ func New() (*Config, error) {
 	}
 	flag.Parse()
 	err := config.loadConfigFromFile()
-	config.setLogLevel()
+
+	if err != nil {
+		return config, err
```
Comment (dankraw) [inline config/config.go]: I think we should return `nil, err` here as config is already defined as `Config{}`

Comment (janisz) [inline config/config.go]: Fixed

**Comment by dankraw** [comment]
:+1:

**Comment by janisz** [comment]
We have Marathon password leak in logs. I fixed it and also make logs more consistent every log (if possible) has `"Id"` field which contains Task ID or (if not exist in current context) Application ID. Also added some more debug logs.

File: config/config.go [hunk 2]
```diff
@@ -68,8 +82,11 @@ func (config *Config) parseFlags() {
 	flag.DurationVar(&config.Metrics.Interval, "metrics-interval", 30*time.Second, "Metrics reporting interval")
 	flag.StringVar(&config.Metrics.Addr, "metrics-location", "", "Graphite URL (used when metrics-target is set to graphite)")
 
+	// Log
+	flag.StringVar(&config.Log.Level, "log-level", "info", "Log level: panic, fatal, error, warn, info, or debug")
+	flag.StringVar(&config.Log.Format, "log-format", "text", "Log format: JSON, text")
```
Comment (dankraw) [inline config/config.go]: in code we check against "TEXT" (uppercase) - we should stick to the rule here.

**Pull request merged**